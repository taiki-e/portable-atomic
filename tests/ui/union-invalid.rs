pub mod multi_fields_no_repr_c {
    use portable_atomic::Atomicable;

    #[derive(Clone, Copy, Atomicable)]
    union A {
        f0: u8,
        f1: u8,
    }
}

fn main() {}
