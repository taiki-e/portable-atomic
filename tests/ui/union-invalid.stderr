error: portable-atomic: union with multiple fields must be #[repr(C)]
 --> tests/ui/union-invalid.rs:5:11
  |
5 |     union A {
  |           ^
